### ISSUE TITLE
TensorFlowServingClient gets 429 errors from tornado when using gen.multi()

### ISSUE THREAD

**Issue by willgraf** [vanvalenlab/kiosk-redis-consumer#3]
The tf-serving client sends too many requests at once (?) and throws a 429 error code when trying to use `gen.multi` to send all requests once.  This should help with some significant performance improvements.  I suspect the `max_clients` may be related to this problem.

**Comment by willgraf** [comment]
Looks like by sending so many requests at once to `tf-serving`, we can cause the tf-serving node to OOM:

```
[ERROR]:[TensorFlowServingClient]: Error: HTTP 429: Client Error: OOM when allocating tensor with shape[16,200,505,505] and type float on /job:localhost/replica:0/task:0/device:GPU:0 by allocator GPU_0_bfc
	 [[Node: conv2d_4/Conv2D = Conv2D[T=DT_FLOAT, data_format="NCHW", dilations=[1, 1, 1, 1], padding="VALID", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:GPU:0"](conv2d_4/Conv2D-0-TransposeNHWCToNCHW-LayoutOptimizer, conv2d_4/kernel/read)]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.
```

This may be a deal-breaker for using `gen.multi`.

**Comment by willgraf** [comment]
`tornado` is removed in #12.  This OOM issue can be resolved by limiting the number of simultaneous clients.  This issue will be closed by #12.

**Comment by willgraf** [comment]
All HTTP Clients have been removed by #16 .

This issue is resolved by #16
